modules/services/swaync.nix

```
@@ -0,0 +1,154 @@
+{ pkgs, lib, config, ... }:
+let
+  cfg = config.services.swaync;
+
+  jsonFormat = pkgs.formats.json { };
+
+  settings = {
+    "$schema" = "${cfg.package}/etc/xdg/swaync/configSchema.json";
+  } // cfg.settings;
+
+  configFile = pkgs.writeTextFile {
+    name = "swaync/config.json";
+    text = builtins.toJSON settings;
+    # TODO uncomment once https://github.com/ErikReider/SwayNotificationCenter/issues/298 fixed
+    # checkPhase = "${pkgs.check-jsonschema}/bin/check-jsonschema --schemafile ${settings."$schema"} $out ";
+  };
+in {
+  meta.maintainers = [ lib.maintainers.rhoriguchi ];
+
+  options.services.swaync = {
+    enable = lib.mkEnableOption "Swaync notification daemon";
+
+    package = lib.mkOption {
+      type = lib.types.package;
+      default = pkgs.swaynotificationcenter;
+      defaultText = lib.literalExpression "pkgs.swaynotificationcenter";
+      description = ''
+        Swaync package to use. Set to <code>null</code> to use the default package.
+      '';
+    };
+
+    systemd = {
+      enable = lib.mkEnableOption "Systemd integration";
+
+      target = lib.mkOption {
+        type = lib.types.str;
+        default = "graphical-session.target";
+        example = "sway-session.target";
+        description = ''
+          Systemd target to bind to.
+        '';
+      };
+    };
+
+    style = lib.mkOption {
+      type = lib.types.nullOr (lib.types.either lib.types.path lib.types.lines);
+      default = null;
+      description = ''
+        CSS style of the bar.
+        </para>
+        <para>
+        See <link xlink:href="https://github.com/ErikReider/SwayNotificationCenter/blob/main/src/style.css"/>
+        for the documentation.
+        </para>
+        <para>
+        If the value is set to a path literal, then the path will be used as the css file.
```

Should be Markdown now.

```suggestion
        CSS style of the bar. See
        <https://github.com/ErikReider/SwayNotificationCenter/blob/main/src/style.css>
        for the documentation.

        If the value is set to a path literal, then the path will be used as the CSS file.
```

---

I had to remove the `<` and `>` from the url to get it to build.